server/rpm/pbench-server.spec.j2

```diff
@@ -22,8 +22,6 @@ Requires: python3-libselinux
 %endif
 
 Requires: npm
-Requires: pbench-dashboard
```

Comment:
If you are going to remove `pbench-dashboard`, you might as well remove `npm` too.  But why not move this to a separate commit?  This commit already has 100+ files in it.

Reply:
> If you are going to remove `pbench-dashboard`, you might as well remove `npm` too.

That's not quite true (I think).  That is, the Dashboard sources are now in the same repo as the Server, and the Dashboard will (quite soon, I hope) be installed by the same RPM as the Server, so, to the extent that the running Dashboard depends on `npm`, this spec file will need a `Requires` for it.

Thus, if you tear it out in this PR, I'll be adding it right back in in an upcoming PR.  🙂

Reply:
Where does the dashboard code depend on `npm` at run time?

Reply:
That's a fair question.  I believe the answer is, "it doesn't".

However, other than "run-time", there's also "build-time" and "install-time".  Someone smarter than me needs to figure out whether we want to "build" the Dashboard when we build the RPM (and then do a straightforward copy-type installation of it), or whether we want to build the Dashboard when we "install" (which will likely make our RPM _much_ smaller, but it will require an Internet connection at install-time, and it's likely to produce less deterministic results).

Given that we intend to deploy the Server (and, therefore, the Dashboard) as a container, we'll control the "install-time" environment, so it may be reasonable to run `npm` at that point instead of during the RPM build.  (For the container deployment, it'll probably be "six of one vs. a half dozen of the other", but the difference will include whether the spec file requires `npm` or not.)

Reply:
> The dashboard `build` script relies on `npm`, but a dashboard deployment doesn't.

That statement is probably too difficult to unpack here.  (I.e., "replies" implies some "present tense", and I'm operating in the "near future"; and, how/when/where the dashboard gets "built" vs. "deployed" is actually open for discussion -- I'm contemplating doing virtually nothing for the Dashboard during the RPM build and then running `npm` when the RPM is installed, so, yeah, `npm` will need to be one of the `Requires`....)

>  I'm happy to remove that, too.

You are welcome to remove it, if it will please Peter and/or improve life for you.  However, I fully expect to put it right back in there, so it'll only be "gone" for a little while.  😀

Reply:
Don't worry, it won't be coming back.  =)  We can have a discussion about how we'll handle deployments in the future.  The browser never makes a reference to files in the npm tree, so we don't need the dependency in the RPM.

Reply:
Right; there's no RPM dependency. If there's a Dockerfile dependency, that's a completely different matter and shouldn't be resolved by burdening the RPM.

Reply:
Well...the RPM can run commands when it is installed, and, for my changes, I was contemplating having it run `npm` then, in which case there _would_ be an RPM dependency.  However, Peter pointed out that there is another approach which is probably better, so I won't be needing this.